packages/hub/services/prisma-extensions/exchange-rate.ts

```diff
@@ -0,0 +1,33 @@
+import { Prisma, PrismaClient } from '@prisma/client';
+import { CryptoCompareConversionBlock } from '../exchange-rates';
+
+type ExchangeRateGetter = Prisma.ExchangeRateDelegate<any>;
+
+export interface ExtendedExchangeRate extends ExchangeRateGetter {
+  select(from: string, to: string[], date: string, exchange: string): Promise<CryptoCompareConversionBlock | null>;
+}
+
+export function getExchangeRateExtension(client: PrismaClient) {
+  return {
+    async select(
+      from: string,
+      to: string[],
+      date: string,
+      exchange: string
+    ): Promise<CryptoCompareConversionBlock | null> {
```

Comment:
I think this would be more readable with a named params style argument:

```ts
    let cachedValue = await prisma.exchangeRate.select({
        from: 'BTC',
        to: ['USD'],
        date: '2022-06-01',
        exchange: DEFAULT_CRYPTOCOMPARE_EXCHANGE
    });
```